crates/astria-composer/src/searcher/tests.rs

```diff
@@ -0,0 +1,17 @@
+mod tests {
+    use ethers::providers::Ws;
+
+    use crate::{
+        config::Config,
+        searcher::Searcher,
+    };
+
+    #[tokio::test]
```

Comment:
I agree that we should replace the `healthz` with `readyz` - this was my intention and it currently has `healthz` because I did not understand the difference between the two.

I like the idea of pinging both the sequencer and geth node to signal readiness. Will do.

Reply:
Implemented and used for spinning up composer in tests